src/test/kotlin/com/pubnub/api/crypto/CryptoModuleTest.kt

```diff
@@ -0,0 +1,143 @@
+package com.pubnub.api.crypto
+
+import com.pubnub.api.crypto.cryptor.AesCbcCryptor
+import com.pubnub.api.crypto.cryptor.Cryptor
+import com.pubnub.api.crypto.cryptor.LegacyCryptor
+import com.pubnub.api.crypto.data.EncryptedData
+import com.pubnub.api.crypto.data.EncryptedStreamData
+import org.junit.Assert
+import org.junit.jupiter.api.Test
+import java.io.InputStream
+
+class CryptoModuleTest {
+
+    @Test
+    fun `can decrypt encrypted message using LegacyCryptoModule with randomIV`() {
+        // given
+        val cipherKey = "enigma"
+        val legacyCryptoModule = CryptoModule.createLegacyCryptoModule(cipherKey)
+        val msgToEncrypt = "Hello world".toByteArray()
+
+        // when
+        val encryptedMsg = legacyCryptoModule.encrypt(msgToEncrypt)
+        val decryptedMsg = legacyCryptoModule.decrypt(encryptedMsg)
+
+        // then
+        Assert.assertTrue(msgToEncrypt.contentEquals(decryptedMsg))
+    }
+
+    @Test
+    fun `can decrypt encrypted message using LegacyCryptoModule with staticIV`() {
+        // given
+        val cipherKey = "enigma"
+        val legacyCryptoModule = CryptoModule.createLegacyCryptoModule(cipherKey)
```

Comment:
I don't see staticIv here

Reply:
Yes. I will change it to:
val legacyCryptoModule = CryptoModule.createLegacyCryptoModule(cipherKey, false)